src/js/modules/supervisors/FileManager.ts

```diff
@@ -54,6 +53,11 @@ class FileManager {
     }
   }
 
+  startWatchers(): void {
+    this.submitDirWatcher = watch(this.submitDir);
+    this.activeDirWatcher = watch(this.activeDir);
```

Comment:
what about shutdown? this.submitDirWatcher.close() ?

Reply:
Oh sorry, I had that change on my stash :facepalm: but yes I have a close method that calls `whatcher.close`

Reply:
Done